question-service/controller/question-controller.js

```diff
@@ -1,115 +1,107 @@
 import { 
-    createQuestion, 
-    findQuestionById, 
-    findByTitle,
-    findAllQuestions,
-    updateQuestionById,
-    deleteQuestionById,
+    createQuestion as importedCreateQuestion, 
```

Comment:
I feel that createQuestion is clearer than importedCreateQuestion, is there any reason for adding “imported” in front of all the functions?

Reply:
Because there is a function called `createQuestion` in this file, and the imported function also has the same name so I added `imported` in front of it.

The same applies to other functions